man/memkind.3

```diff
@@ -688,6 +688,57 @@ The available kinds of memory:
 .B MEMKIND_DEFAULT
 Default allocation using standard memory and default page size.
 .TP
+.B MEMKIND_HIGHEST_CAPACITY
+Allocate from a NUMA node(s) that has the highest capacity among all nodes in the system.
+.TP
+.B MEMKIND_HIGHEST_CAPACITY_PREFERRED
+Same as
```

Comment:
"Same as above" or "Same as MEMKIND_HIGHEST_CAPACITY"

Reply:
It is "Same as MEMKIND_HIGHEST_CAPACITY" see https://github.com/memkind/memkind/blob/46e6b883665a9eda651f4ee775c214282dbe6651/man/memkind.3#L694-L696